src/new_index/schema.rs

```diff
@@ -273,13 +280,53 @@ impl Indexer {
         let tip = daemon.getbestblockhash()?;
         let new_headers = self.get_new_headers(&daemon, &tip)?;
 
+        // Must rollback blocks before rolling forward
+        let headers_len = {
+            let mut headers = self.store.indexed_headers.write().unwrap();
+            let reorged = headers.apply(new_headers.clone());
+            assert_eq!(tip, *headers.tip());
+            let headers_len = headers.len();
+            drop(headers);
+
+            if !reorged.is_empty() {
+                if reorged.len() > 10 {
+                    warn!("reorg of over 10 blocks detected! Wonky stuff might happen!");
+                }
```

Comment:
is there anything special about 10 blocks here, or is it just a arbitrary threshold for "damn that's a big reorg"?

Reply:
arbitrary